doc/tutorials/12-constant_pH/si_units.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+# physical constants
+import scipy.constants
+#as const
+
+class SIunits():
```

Comment:
actually there already exists a package that may be doing what you are looking for:
https://pint.readthedocs.io/

Reply:
*e.g.*:
``` python
#!/usr/bin/env python3

import scipy.constants
import pint

ureg = pint.UnitRegistry()

sigma = 5 * ureg.nanometer
tau = 1.0 * ureg.picosecond

c = 0.1 * ureg.mol / ureg.litre
box_volume = 10 * sigma * 20 * sigma * 5 * sigma
N_part = int((c * box_volume).to(ureg.mole).magnitude * scipy.constants.N_A)
```

Reply:
DONE